CMakeLists.txt

```diff
@@ -127,6 +127,15 @@ set(libsrc
     src/printer_tree.c
     src/plugins.c
     src/plugins_types.c
+    src/plugins_exts.c
+    src/plugins_exts/metadata.c
+    src/plugins_exts/nacm.c
+    src/plugins_exts/yangdata.c
+    src/xml.c
+    src/xpath.c
+    src/validation.c)
+
+set(type_plugins
```

Comment:
Wouldn't be better to add the `${type_plugins}` into `libsrc`? The build system expects the files there, so instead of searching where the `${type_plugins}` should be used, just add the list into the original variable.

Reply:
Maybe also somehow describe/note, that the files are separated just because of generating documentation from them, could be useful.

Reply:
Okay, fixed.